Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -549,15 +549,11 @@ class BrowserViewController: UIViewController {
     }
 
     fileprivate func hasPendingCrashReport() -> Bool {
```

Comment:
Nit: We might as well just remove this method and use `KSCrash.sharedInstance().crashedLastLaunch` directly